apps/accounts/forms.py

```diff
@@ -377,8 +393,38 @@ class NetworkFootprintForm(MultiModelForm):
     form_classes = {
         "ips": IpRangeFormset,
         "asns": AsnFormset,
+        "extra": ExtraNetworkForm
     }
 
+    def clean(self):
+        """
+        Validate that a form at least has:
+        1. some IP and/or AS data
+        2. or some explanation
+
+        """
+        super().clean()
+
+        # fetch our forms
+        ip_form, asn_form, explanation_form = [form for form in self.forms.values()]
```

Comment:
I'm not sure if `self.forms` respects the ordering, maybe we shouldn't rely on it. It would be better to get each form by the key